### PR TITLE
Delete workspace

### PR DESCRIPTION
**Pull request by dprincecoder**
# Fixes Issue/Linear Ticket
https://linear.app/team-brainbox/issue/FRO-200/zuri-chat

**My PR Closes #issue_number_here** or
**My PR Fixes ID-linear_ticket_number_here**
FRO-200/Zuri-chat

# Changes proposed

## What were you told to do?

As Admin, I should be able to delete a workspace/organization

## What did you do?

I set up the routing for a delete confirmation page
I handle the API endpoint for deleting a workspace, the admin is currently in, using the workspace id.

# Check List (Check all the applicable boxes)

🚨Please review the [style guide for contributing](https://github.com/zurichat/zc_main/blob/dev/docs/STYLING.md) and [guidelines for contributing](https://github.com/zurichat/zc_main/blob/dev/docs/CONTRIBUTING.md) to this repository.

- [x] My code follows the code style of this project.
- [x] This PR does not contain plagiarized content.
- [x] The title and description of the PR are clear and explain the approach.
- [x] I am making a pull request against the **dev branch** (left side).
- [x] My commit messages styles match our requested structure.
- [x] My code additions will fail neither code linting checks nor unit tests.
- [x] I am only making changes to the files I was requested to.

# Screenshots/ Videos

### CODE REVIEW

File: packages/main/src/components/delete-workspace/deleteWorkspace.js [hunk 1]
```diff
@@ -0,0 +1,157 @@
...
+    await axios
+      .delete(
+        `https://staging.api.zuri.chat/organizations/${currentWorkspace}`,
+        {
+          headers
+        }
+      )
+      .then(res => {
+        if (res.status === 200) {
+          console.log("Workspace deleted successfully");
```
Comment (JulianaSau) [inline packages/main/src/components/delete-workspace/deleteWorkspace.js:28]: remove console log. Use a toast notification instead

File: packages/main/src/components/delete-workspace/deleteWorkspace.js [hunk 2]
```diff
@@ -0,0 +1,157 @@
...
+          localStorage.removeItem(currentWorkspaceShort);
+          localStorage.removeItem(orgName);
+          localStorage.removeItem(currentWorkspace);
+          window.location.href = "/create-workspace";
```
Comment (JulianaSau) [inline packages/main/src/components/delete-workspace/deleteWorkspace.js:32]: Why not use react-router to route to this page?

File: packages/main/src/components/delete-workspace/deleteWorkspace.js [hunk 3]
```diff
@@ -0,0 +1,157 @@
...
+        }
+      })
+      .catch(err => {
+        console.log(err);
```
Comment (JulianaSau) [inline packages/main/src/components/delete-workspace/deleteWorkspace.js:36]: instead of console.log, show a toast notification

**Review by JulianaSau — CHANGES_REQUESTED** [review]
Resolve the conflicts yu have and implement the changes requested

**Comment by dprincecoder** [comment]
resolved

File: packages/main/src/pages/protected/manage-workspace/ManageWorkspace.jsx [hunk 4]
```diff
@@ -204,8 +204,8 @@ const adminSettings = () => {
                 </div>
               </div>
               <div className={classes.floatright}>
-                <Link to="/admin/delete">
-                  <button>{t("delwork_manage")}</button>
+                <Link to="/admin/workspace/delete">
+                  <button>Delete Workspace</button>
```
Comment (JulianaSau) [inline packages/main/src/pages/protected/manage-workspace/ManageWorkspace.jsx:208]: Someone adds translation to the app and you remove it? Isn't this counter productive. If you don't understand something just leave it there

File: packages/main/src/pages/protected/manage-workspace/ManageWorkspace.jsx [hunk 5]
```diff
@@ -1,26 +1,26 @@
-import React, { useEffect, useState } from "react";
 import { AnimateSharedLayout, motion } from "framer-motion";
+import React, { useEffect, useState } from "react";
```
Comment (JulianaSau) [inline packages/main/src/pages/protected/manage-workspace/ManageWorkspace.jsx:2]: Why move the react import to the second line when it's supposed to be at the very beginning in a js file

**Review by JulianaSau — CHANGES_REQUESTED** [review]

File: packages/main/src/components/delete-workspace/deleteWorkspace.js [hunk 6]
```diff
@@ -0,0 +1,151 @@
...
+  const token = userData.token;
+
+  const deleteWorkspace = async e => {
+    e.preventDefault();
+    const headers = {
+      "Content-Type": "application/json",
+      Authorization: `Bearer ${token}`
+    };
+    const res = await axios.delete(
+      `https://api.zuri.chat/organizations/${currentWorkspace}`,
```
Comment (JulianaSau) [inline packages/main/src/components/delete-workspace/deleteWorkspace.js:20]: It's not good to reference the API you're calling here
use the base url constant provided in the app
it should be coming from @zuri-utilities

**Review by JulianaSau — CHANGES_REQUESTED** [review]

File: packages/main/src/pages/protected/manage-workspace/ManageWorkspace.jsx (hunk 5 above)
Comment (dprincecoder) [reply packages/main/src/pages/protected/manage-workspace/ManageWorkspace.jsx:2]: I didnt import react or made use of useEffect or useState at all @JulianaSau is from somelse merge, or is prettier that organise it like so

File: packages/main/src/components/delete-workspace/deleteWorkspace.js (hunk 6 above)
Comment (dprincecoder) [reply packages/main/src/components/delete-workspace/deleteWorkspace.js:20]: I didnt't know the api that base_url belongs to so i have to make sure ap.zuri.chat is what I'm working with. Buh nevertheless i will change it

File: packages/main/src/pages/protected/manage-workspace/ManageWorkspace.jsx (hunk 4 above)
Comment (dprincecoder) [reply packages/main/src/pages/protected/manage-workspace/ManageWorkspace.jsx:208]: I will change it, that was someone merge stuff to counter my work which i added earlier, so i had to keep what i was working with earlier, by the way, this button wasn't there I'm the one that added it, only for me to pull and find out someone somewhere is also changing stuff.

File: packages/main/src/components/delete-workspace/deleteWorkspace.js (hunk 6 above)
Comment (JulianaSau) [reply packages/main/src/components/delete-workspace/deleteWorkspace.js:20]: It changes depending on the environment the app is being run in. So it's fine

File: packages/main/src/pages/protected/manage-workspace/ManageWorkspace.jsx (hunk 5 above)
Comment (JulianaSau) [reply packages/main/src/pages/protected/manage-workspace/ManageWorkspace.jsx:2]: React should always be at the top level

File: packages/main/src/pages/protected/manage-workspace/ManageWorkspace.jsx (hunk 4 above)
Comment (JulianaSau) [reply packages/main/src/pages/protected/manage-workspace/ManageWorkspace.jsx:208]: Well, translation for all pages is ongoing. When resoling conflicts, you accept both changes and keep the changes from the other person and add the ones you're working on. Not removing the other person's work entirely. The only instance when you can delete is if that functionality isn't needed anymore

**Comment by dprincecoder** [comment]
@JulianaSau I have fixed requested changes

**Review by JulianaSau — APPROVED** [review]

**Pull request merged**